nvi-commons/src/main/java/no/sikt/nva/nvi/common/db/CandidateRepository.java

```diff
@@ -161,52 +163,52 @@ public List<DbNote> getDbNotes(UUID candidateIdentifier) {
     }
 
     public List<NoteDao> getNotes(UUID candidateIdentifier) {
-        return noteTable.query(queryCandidateParts(candidateIdentifier, NoteDao.TYPE))
-                   .items()
-                   .stream()
-                   .toList();
+        return noteTable.query(queryCandidateParts(candidateIdentifier, NoteDao.TYPE)).items().stream().toList();
     }
 
     public void updateCandidateAndRemovingApprovals(UUID identifier, CandidateDao nonApplicableCandidate) {
         candidateTable.putItem(nonApplicableCandidate);
         var approvalStatuseDaos = getApprovalStatuses(identifier);
         if (!approvalStatuseDaos.isEmpty()) {
             var transactionBuilder = TransactWriteItemsEnhancedRequest.builder();
-            approvalStatuseDaos.forEach(approvalStatusDao ->
-                                            transactionBuilder.addDeleteItem(approvalStatusTable, approvalStatusDao));
+            approvalStatuseDaos.forEach(
+                approvalStatusDao -> transactionBuilder.addDeleteItem(approvalStatusTable, approvalStatusDao));
             client.transactWriteItems(transactionBuilder.build());
         }
     }
 
+    public void updateCandidateOnly(CandidateDao candidate) {
```

Comment:
Rename method to only `update`?

Reply:
I want method name that says that we are updating candidateOnly, cause other method updates also approvals and notes.

Reply:
Cause other method has name -> updateCandidate()

Reply:
You can have several methods with the same name, but with different input parameters?